RecoBTag/CTagging/python/RecoCTagging_cff.py

```diff
@@ -0,0 +1,17 @@
+from RecoBTag.CTagging.pfInclusiveSecondaryVertexFinderCvsLTagInfos_cfi import *
```

Comment:
One of the features of the unscheduled mode is that it will run any loaded modules whose names match the required input products. For b-tagging in particular, this essentially means that when PAT is run, for example, to produce MiniAOD, all required b-tag discriminators will be re-run and not taken from RECO/AOD. This is good for cases where one wants to run the latest version of the taggers and ignore discriminators already stored in the event.

So what I don't understand is why would the PAT tests fail. Re-building TagInfos from AOD should not cause any failures.